src/Microsoft.AspNet.Mvc.Core/OptionDescriptors/ModelBinderDescriptorExtensions.cs

```diff
@@ -80,5 +80,24 @@ public static class ModelBinderDescriptorExtensions
             descriptors.Insert(index, descriptor);
             return descriptor;
         }
+
+
+        /// <summary>
+        /// Removes all instances of <typeparamref name="TInstance"/> from a list of ModelBinderDescriptors.
+        /// </summary>
+        /// <typeparam name="TInstance">A type that represents an <see cref="IModelBinder"/>.</typeparam>
+        /// <param name="descriptors">A list of ModelBinderDescriptors.</param>
```

Comment:
At line 86, to be consistent with other methods, 'list of ModelBinderDescriptors' should be changed to 'descriptor collection'?